packages/story-editor/src/app/pageDataUrls/test/usePageDataUrls.js

```diff
@@ -60,19 +60,19 @@ describe('usePageDataUrls', () => {
     mockIdleCallbacks = [];
     runIdleCallbacks = () => {
       while (mockIdleCallbacks.length > 0) {
-        const [, callback] = mockIdleCallbacks.shift();
-        callback();
+        const { task } = mockIdleCallbacks.shift();
```

Comment:
Just noting that converting this test to TypeScript could help with these changes here, making the test more robust

Reply:
Should we convert the tests in these PRs?